onyo/cli/rmdir.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from onyo.lib.onyo import OnyoRepo
+from onyo.lib.commands import onyo_rmdir
+from onyo.lib.inventory import Inventory
+from onyo.shared_arguments import (
+    shared_arg_message,
+    shared_arg_no_auto_message,
+)
+
+if TYPE_CHECKING:
+    import argparse
+
+args_rmdir = {
+    'directory': dict(
+        metavar='PATH',
+        nargs='+',
+        help=r"""
+            Directories to delete; or assets to convert into an Asset File.
+        """
+    ),
+    'message': shared_arg_message,
+    'no_auto_message': shared_arg_no_auto_message,
+}
+
+epilog_rmdir = r"""
+.. rubric:: Examples
+
+Remove a user from a group:
+
+.. code:: shell
+
+    $ onyo rmdir accounting/Bingo\ Bob/
+
+Convert an empty Asset Directory into an Asset File:
+
+.. code:: shell
+
+    $ onyo rmdir accounting/Bingo\ Bob/laptop_apple_macbook.oiw629/
+
+"""
+
+
+def rmdir(args: argparse.Namespace) -> None:
+    r"""
+    Delete empty **DIRECTORY**\ s or convert empty Asset Directories into Asset
```

Comment:
Does it work on dirs without an anchor file, i.e. actual empty dirs, or only if tracked with onyo?

Reply:
It errors:
```
❱ mkdir tst
❱ onyo rmdir tst 
ERROR: Not an inventory directory: /home/aqw/git/inm7/inventory/tst
```

Which IMO is the right thing to do. `onyo mkdir` adds a directory to the inventory; it doesn't cares if it already exists. However, `rmdir` is about removing a directory from the inventory. It makes sense if it errors and days "I know know about this thing; failing".

Reply:
Yes, perfect. Thank you!